src/lib/VictoryConditon.ts

```diff
@@ -0,0 +1,37 @@
+import { SimulatorState } from './SimulatorState';
+
+export interface VictoryCondition {
+    name: string;
+    description: string;
+    isMet(simulatorState: SimulatorState): boolean;
+}
+
+export const TimeVictory: VictoryCondition = {
+    name: 'Time victory',
+    description: 'You managed to survice one year!',
+    isMet: (simulatorState: SimulatorState) => simulatorState.currentState.days >= 365
+};
+
+export const RaceToZero: VictoryCondition = {
+    name: 'Zero infections in the last 30 days',
+    description: 'Infections are at zero for the past 30 days.',
+    isMet: (simulatorState: SimulatorState) => {
+        if (simulatorState.currentState.days < 30) {
+            return false;
+        } else {
+            const cutOff = simulatorState.currentState.days - 30;
```

Comment:
This one is a bit more complicated, and I'm not sure this is a valid condition, so we will need to discuss this. It goes back in the game history until a cut off point (30 days prior to the current game date) to check if during that period the number of infected remains zero. 

I say this might be impossible because of the inherent randomness of the simulator and the imported cases. We will need to check if the "zero" is in fact a small delta.